include/prefetched_buffer.h

```diff
@@ -0,0 +1,74 @@
+#pragma once 
```

Comment:
Maybe run this file through `clang-format`..? :)

Reply:
Will do!